ruleset/rules/0515-exim_rules.xml

```diff
@@ -1,69 +1,73 @@
 <!--
-  -  Exim rules
-  -  Author: Alexandr Garaga.
-  -  Updated by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  Copyright (C) 2009 Trend Micro Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
+<!--
+  Exim 4 rules 
+-->
+
+<!-- ID: 87500 - 87600 -->
+
+
 <group name="exim,">
-    <rule id="87500" level="0">
-      <decoded_as>windows-date-format</decoded_as>
-      <regex>^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d SMTP </regex>
-      <description>Exim SMTP Messages Grouped.</description>
-    </rule>
 
-    <rule id="87501" level="0">
-      <decoded_as>windows-date-format</decoded_as>
-      <regex>^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d dovecot</regex>
-      <description>dovecot messages grouped.</description>
-    </rule>
+  <rule id="87500" level="0">
+    <decoded_as>windows-date-format</decoded_as>
+    <regex>^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d SMTP </regex>
+    <description>Exim: SMTP Messages Grouped.</description>
+  </rule>
+
+  <rule id="87501" level="0">
+    <decoded_as>windows-date-format</decoded_as>
+    <regex>^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d dovecot</regex>
+    <description>dovecot messages grouped.</description>
+  </rule>
+
+  <rule id="87502" level="5">
+    <if_sid>87501</if_sid>
+    <match>authenticator failed</match>
+    <description>Exim: Auth failed</description>
+    <group>invalid_login,authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,</group>
+  </rule>
 
-    <rule id="87506" level="5">
-      <if_sid>87501</if_sid>
-      <match>authenticator failed</match>
-      <description>Exim Auth failed</description>
-      <group>invalid_login,authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,</group>
-    </rule>
+  <rule id="87503" level="10" frequency="8" timeframe="240">
+    <if_matched_sid>87502</if_matched_sid>
+    <same_source_ip />
+    <description>Exim: brute force attack (multiple auth failures).</description>
+    <mitre>
+      <id>T1110</id>
+    </mitre>
+    <group>authentication_failures,pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_11.4,gdpr_IV_35.7.d,gdpr_IV_32.2,</group>
```

Comment:
Groups are not correctly ordered.

Reply:
Fixed: https://github.com/wazuh/wazuh/pull/11344/commits/9a59070843cd7f2ad2bc7bbd31de592f0f469607